nilearn/glm/tests/test_first_level.py

```diff
@@ -14,7 +14,8 @@
 
 from nilearn._utils.data_gen import (create_fake_bids_dataset,
                                      generate_fake_fmri_data_and_design,
-                                     write_fake_fmri_data_and_design)
+                                     write_fake_fmri_data_and_design,
+                                     basic_paradigm)
```

Comment:
You need to handle the definition of `basic_paradigm` line 475 in this file.

Reply:
Oops missed that one, thanks!